### DISCUSSION TITLE
v0.12.1

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/steamguard-cli#322] [General]
## What's Changed
* remove `normalize_comments` rustfmt rule by @dyc3 in https://github.com/dyc3/steamguard-cli/pull/316
* fix how UserLogin handles access tokens for new api behavior, fixes #319 by @dyc3 in https://github.com/dyc3/steamguard-cli/pull/320
* misc fixes for deb packaging by @dyc3 in https://github.com/dyc3/steamguard-cli/pull/315

**Full Changelog**: https://github.com/dyc3/steamguard-cli/compare/v0.12.0...v0.12.1

<hr /><em>This discussion was created from the release <a href='https://github.com/dyc3/steamguard-cli/releases/tag/v0.12.1'>v0.12.1</a>.</em>

**Comment by mahdikhaligh** [comment]
### update without remove app
for get new version need remove old version and download new version and install again and import mafile account ...?

**Reply by dyc3** [reply]
You just replace the old version with the new version. No need to import mafiles or anything, it should just work.
